easybuild/easyconfigs/n/NGSadmix/NGSadmix-32-foss-2018b.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CmdCp'
+
+name = 'NGSadmix'
+version = '32'
+
+homepage = 'http://www.popgen.dk/software/index.php/NgsAdmix'
+description = """This page contains information about the program called 
+NGSadmix, which is a very nice tool for finding admixture proportions 
+from NGS data. It is based on genotype likelihoods."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
Since it does not seem to require MPI, ScaLAPACK or FFTW, how about moving it down to the `GCC` toolchain? (`GCCcore` might be a little too extreme since different compilers could affect performance and correctness differently)

Reply:
@vanzod Thanks for pointing this out to me. When I done that I was new and less experienced than I am now. With each problem you face you learn a bit more. So these comments are helpful for me.

Reply:
@sassy-crick No worries, your contribution is very important to us